Source/SpectrumAnalysis/KTChannelAggregator.hh

```diff
@@ -0,0 +1,119 @@
+/*
+ * KTChannelAggregator.hh
+ *
+ *  Created on: Jan 25, 2019
+ *      Author: P. T. Surukuchi
+ */
+
+#ifndef KTCHANNELAGGREGATOR_HH_
+#define KTCHANNELAGGREGATOR_HH_
+
+#include "KTProcessor.hh"
+#include "KTData.hh"
+#include "KTPowerSpectrumData.hh"
+#include "KTChannelAggregatedData.hh"
+#include "KTFrequencySpectrumDataFFTW.hh"
+#include "KTAxisProperties.hh"
+
+#include "KTMemberVariable.hh"
+
+#include "KTSlot.hh"
+
+#include "KTMath.hh"
+
+namespace Katydid
+{
+
+    KTLOGGER(avlog_hh, "KTChannelAggregator.hh");
+
+    class KTChannelAggregatedData;
+    class KTPowerSpectrumData;
+
+    /*
+     @class KTChannelAggregator
+     @author P. T. Surukuchi
+     
+     @brief Multiple channel summation for Phase-III and IV
+     
+     @details
+     More details to come.
+     
+     Configuration name: "channel-aggregator"
+     
```

Comment:
Missing the documentation of the configuration values

Reply:
Fixed in commit ae569a85bc54b10279dfe0c08acefb98f8c59bd2